docs/README.md

```diff
@@ -1,58 +1,68 @@
 
-We use `yarn` as our default package manager, before coding make sure to run yarn on: `root` and `react` folders.
+![image](https://user-images.githubusercontent.com/8517023/73387868-f1b36f80-42af-11ea-8e24-3045d2c819b4.png)
 
-## Some features:
+## Configuration
 
-### Tests
+1. Add `store-link` app to your theme's dependencies in the `manifest.json`, for example:
 
-For testing we use `@vtex/test-tools`, our own testing framework based on `react-testing-library`, the tests should be located on the `react/__tests__` folder. For references, visit our [repository](https://github.com/vtex/test-tools).
-
-### Hooks
-
-Husky hooks tha runs on every `pre-commit` and `pre-push`.
-
-### Intl Equalizer
-
-Tool for equalizing the messages located on the `messages` folder/builder. It's configured to use the **en.json** as the default file for comparison. For references, visit our [repository](https://github.com/vtex/intl-equalizer).
-
-### Lint + Formatting
-
-TS lint configured with Prettier and .Config.
+```jsonc
+{
+  "vtex.store-link": "0.x"
+}
+```
 
-### Available Scripts
+2. Now you can use the blocks exported by the app. In the following example you can see the usage of the `link.product` inside of a quick view component:
 
-```json
+```jsonc
 {
-  "lint": "cd ./react && yarn lint",
-  "test": "cd ./react && yarn test",
-  "lint:locales": "intl-equalizer",
-  "locales:fix": "intl-equalizer --fix",
-  "verify": "yarn lint && yarn lint:locales && yarn test"
+  "link.product#product-page": {
+    "props": {
+      "href": "/{slug}/p",
+      "label": "Mais detalhes >"
+    }
+  },
+  "product-summary.shelf": {
+    "children": [
+      "product-summary-image",
+      "product-summary-name",
+      "product-rating-inline",
+      "product-summary-space",
+      "product-summary-price",
+      "link.product#product-page
+    ]
+  },
 }
 ```
 
-### Ci
+You can see that there is a `{slug}` placeholder being passed to the `href` prop. When rendered, the placeholder will be replaced with the respective value of the closest product context and generate a link like `/everyday-necessaire/p`. However, keep in mind that in order for this work, you have to place the `link.product` block inside of a another block that provides a product context, such as the [`ProductSummary`](https://vtex.io/docs/components/product/vtex.product-summary).
 
-#### Install:
+### Props
 
-```yml
-install:
-  commands:
-    - echo Installing Packages...
-    - cd react
-    - npm install
-    - echo Packages installed!
-```
+The `link.product` and `link` blocks has the same props.
 
-#### Pre-build:
+| Prop name | Type     | Description                             | Default value |
+| --------- | -------- | --------------------------------------- | ------------- |
+| `label`   | `string` | The label that you be shown to the user | `undefined`   |
+| `href`    | `string` | The actual link                         | `'#'`         |
```

Comment:
`label` -> Link text
`link` -> Link URL

Reply:
I didnt understand what `#` is :(

Reply:
`undefined` is for a default value that does not exist? We can apply a default description for that. Previously, I was using just a -, but now Im afraid that people misunderstood this - as a default value haha. We could use a  sentence more clarified, such as: "No default value is defined"

What do you think?

Reply:
> `undefined` is for a default value that does not exist? We can apply a default description for that. Previously, I was using just a -, but now Im afraid that people misunderstood this - as a default value haha. We could use a sentence more clarified, such as: "No default value is defined"
> 
> What do you think?

`undefined` is a value in javascript that means exactly what you said, so I think is ok to let `undefined`. 

The default value ` #` means that it will add just a hash to the url if you click, which means that nothing will happen, is kinda a default value of links in general